src/index.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App';
+import 'bootstrap/dist/css/bootstrap.css';
 
```

Comment:
Let's initialize the store here.

Kindly follow the steps in the screenshot below.

![image](https://user-images.githubusercontent.com/31897434/128092065-6b222a03-4d0a-45d3-b1db-a992202d98a6.png)

Reply:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4) instead of `Math.random` function.

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'YDKJS', category: 'Learning'}
```

Reply:
I initialize it in reducers/index to implement the redux-toolkit recommended by redux. This can be seen using redux-dev-tools as a better understanding.